modules/miner/miner.go

```diff
@@ -16,9 +16,15 @@ const (
 
 	// headerForWorkMemory is the number of previous calls to 'headerForWork'
 	// that are remembered.
-	headerForWorkMemory = 50
+	headerForWorkMemory = 20
+
+	// headersPerBlockMemory is the number of headers created by changing the
+	// random transaction of a block before updating the block to contain any
+	// new transactions
+	headersPerBlockMemory = 5
```

Comment:
I made it low for testing. I didn't mean to commit such a low number. What should I change it too?
Maybe 500 and 20? so it will store up to 25 blocks?